package.json

```diff
@@ -15,7 +15,7 @@
     }
   },
   "scripts": {
-    "test": "jest",
+    "test": "jest --coverage",
```

Comment:
check if we need to add an argument to force-exit after run the test

Reply:
I tested it on few projects but did not catch the case when there was a problem with exit